vividus/src/main/resources/org/vividus/spring-vividus.xml

```diff
@@ -18,7 +18,7 @@
     </bean>
 
     <bean class="org.vividus.bdd.issue.KnownIssueProvider" init-method="init">
-        <property name="fileName" value="${known-issue-provider.fileName}" />
+        <property name="fileName" value="${known-issues.file-name}" />
```

Comment:
I would prefer to use the old naming, since `file-name` is not an attribute of `known-issues`